src/schema/v2/image/cropped.ts

```diff
@@ -33,21 +34,24 @@ export const croppedImageUrl = (
     width,
     height,
     quality = DEFAULT_SRCSET_QUALITY,
-  }: CroppedImageArguments
+  }: CroppedImageArguments,
+  { imageService = "gemini" } = {}
```

Comment:
The reason why this 'works' (as in, how this value is accessible here and analogously in the `resized` file), is that this method is actually a resolver. These are defined in separate fields and then imported and used in the actual schema definitions so it's not obvious, but these are resolvers, and so the last argument (which wasn't being used), is the GraphQL context which contains a ton of stuff: loaders, plus the request-specific things like this value from the header.